### DISCUSSION TITLE
who-create-product-and-who-create-change/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#180] [Blog comments]
# who-create-product-and-who-create-change/

뛰어난 기술과 수 없이 많은 서비스가 우리 곁에 놓여진 세상이다. 이제는 누구나 쉽게 제품을 만들 수 있고, 누구나 쉽게 서비스를 이용할 수 있다.  기업은 자신들의 제품을 자랑하고 싶어 하고, 엔지니어들은 자신들의 기술력을 뽐내고 싶어 한다. 그러나 뛰어난 제품, 기술력과 부단한 마케팅에도 불구하고 소수의 선택 받은 기업, 엔지니어를 제외하곤 시장에서 별다른 관심을 갖지 못한다.  

https://hyeon9mak.github.io/who-create-product-and-who-create-change/

**Comment by dhslrl321** [comment]
생각해볼 거리가 많은 이야기네요. 정말 어려운 주제인 것 같기도 하고요.
